src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -342,7 +330,7 @@ public void run() {
     }
 
     int token;
-    ImmutableList.Builder<AccessorView> accessors = ImmutableList.builder();
+    ImmutableList.Builder<String> accessors = ImmutableList.builder();
```

Comment:
It might be worth noting in a comment here and in the function that returns these that these are not just the field names but include the language-specific syntax necessary (`.`, `->`, etc)